### PR TITLE
Added mountPoints on container definition

### PR DESCRIPTION
**Pull request by nutellinoit**
## what
* Add mount points on container definition

## why
* Mount EBS volume into ECS containers (does not work with Fargate)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -94,3 +94,16 @@ variable "log_options" {
     "awslogs-stream-prefix" = "default"
   }
 }
+
+variable "mount_points" {
+  type        = "list"
+  description = "Container mount points. This is a list of map, each map should contain containerPath and sourceVolume"
```
Comment (osterman) [inline variables.tf]:
```suggestion
  description = "Container mount points. This is a list of maps, where each map should contain a `containerPath` and `sourceVolume`"
```

**Comment by pecigonzalo** [comment]
How would this be passed to the `service` tho?

**Comment by pecigonzalo** [comment]
Ok, so you will have to keep the `aws_ecs_task_definition` outside of a module

**Comment by nutellinoit** [comment]
> Ok, so you will have to keep the `aws_ecs_task_definition` outside of a module

yes, i think that this module is only for the container definition

**Comment by pecigonzalo** [comment]
Yes, it is, but I was thinking of how it will be used

**Pull request merged**